### ISSUE TITLE
How to check that I have configured it successfully?

### ISSUE THREAD

**Issue by FlyerMaxwell** [mununum/ns3-gym-csmarl#3]
I think it's better to pull a new issue.

Hi, I have some trouble trying this project. I am not familiar with docker so I configure it with my own PC. I wonder wheather I  have configured the environment successfully. I turn to branch 'develop' and configure the environment step by step with your instruction. When I run './scratch/opengym/simple_test.py', I got the following response. Have I configure it successfuly?

![0971a6eeef12e25fb2a9b962b3b3653](https://user-images.githubusercontent.com/37821095/139805481-73251ac1-072e-4110-8694-2f8335650492.jpg)

Besides, what is the tensorflow version have you used?
When I use tensorflow 2.6.1, the following problem encountered. It seems that you have used tensorflow 1.X.
```
ModuleNotFoundError: No module named 'tensorflow.contrib'.
```

_Originally posted by @FlyerMaxwell in https://github.com/mununum/ns3-gym-csmarl/issues/1#issuecomment-957173269_

**Comment by FlyerMaxwell** [comment]
Additionally, when I try the use case Cognitive Radio in  /scratch/interference-pattern/cognitive-agent-v1.py with tensorflow 1.15, I got the following response.
```
exception at close Operation cannot be accomplished in current state.
```

![43c78bfc5faca1e8243f5c4b04cf4ea](https://user-images.githubusercontent.com/37821095/139814385-0091429e-6fc1-4bee-98e4-b44a4ff0605b.jpg)
![ad315e5be55b343f3dc2951b1a86315](https://user-images.githubusercontent.com/37821095/139812813-3bc39e62-2974-4e77-a99f-f0a9affbb866.jpg)

However, the right response should be:
![a80a7d11d2cea156baddb220400e070](https://user-images.githubusercontent.com/37821095/139812901-99516aa9-a725-4659-84b2-e21d28884a43.jpg)

My configuration:
```
Python=3.6.13
tensorflow-gpu = 1.15
ray=0.8.6
keras = 2.6.0
gcc = 7.5.0
g++ = 7.5.0
```

**Comment by mununum** [comment]
For the first question, you have configured it right. The output on the figure is expected.
For second question, I didn't personally run the cognitive radio example myself.
I think you have to dive in and debug the exception in code level.

**Comment by FlyerMaxwell** [comment]
> For the first question, you have configured it right. The output on the figure is expected. For second question, I didn't personally run the cognitive radio example myself. I think you have to dive in and debug the exception in code level.

Thanks for your reply.  I fixed it by configure it on another ubuntu 16.04 computer. 

Besides, based on your hardware (i9-9900X, 128GB RAM, 2 TITAN Xp GPUs), how long did it take you to train G（10，0.3）? Have you tried to train it with a network with more nodes, such as dozens or hundreds nodes?

**Comment by mununum** [comment]
It took about one week of full training G (10, 0.3) model. As you can see in the paper, I have trained G(N,d) with varying range of N \in [2, 20] and d \in [0, 1].
